packages/keystone/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ListPage/index.tsx

```diff
@@ -436,14 +438,18 @@ function DeleteManyButton({
     useMemo(
       () =>
         gql`
-  mutation($where: [${list.gqlNames.whereUniqueInputName}!]!) {
-    ${list.gqlNames.deleteManyMutationName}(where: $where) {
-      id
-    }
-  }
+        mutation($where: [${list.gqlNames.whereUniqueInputName}!]!) {
+          ${list.gqlNames.deleteManyMutationName}(where: $where) {
+            id
+            ${list.labelField}
+          }
+        }
 `,
       [list]
-    )
+    ),
+    {
+      errorPolicy: 'all',
```

Comment:
[errorPolicy set to all](https://www.apollographql.com/docs/react/data/error-handling/#graphql-error-policies), so we can get handle scenarios where partial deletion success occurs.